tools/custom-organization-policy-library/README.md

```diff
@@ -24,16 +24,24 @@ Configuration specific to an organization such as organization id, bundles to en
 
 Example of values.yaml
 ```
-organization: '3333334'
+organization: '11111111'
 bundles:
-  pci-dss: false
-  cis: false
+ pci-dss: false
+ cis: false
+dryrun: false
+compute:
+ computeAllowedInstanceMachineTypes:
+   params:
+     machine_types:
+     - "n2-standard-1"
+     - "n2-standard-2"
 ```
 
 Once you have initialized a library, you might want to save it to [git](./docs/user_guide.md#https://github.com/GoogleCloudPlatform/policy-library/blob/master/docs/user_guide.md#get-started-with-the-policy-library-repository).
```

Comment:
Can we remove that part it does not make sense. Also the link is to other project